### DISCUSSION TITLE
time-lapse setting in older versions

### DISCUSSION THREAD

**Discussion by ShabnamShadloo** [gopro/labs#325] [Q&A]
Hi all.
I am planning to buy a GoPro and need to know about the timelapse setting. Can older versions of GoPro (mainly 3 or 4) support less frequent timelapse photos for several days or weeks? Say six times a day for a week.

**Comment by dnewman-gpsw** [comment]
No.  There is no Labs firmware for cameras that old.  Cameras with some support would be HERO7 onward.
